apps/condo/domains/miniapp/utils/b2bAppServiceUserAccess/schema.utils.js

```diff
@@ -113,6 +133,10 @@ const generatePermissionFields = ({ config }) => {
         }
     }
 
+    for (const permissionFieldName of allServicePermissionFieldNames) {
+        permissionFields[permissionFieldName] = PERMISSION_FIELD
```

Comment:
schemaDoc?

Reply:
Not needed here. Created above only for read only permission field and contains the explanation what means read only.